hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/replication/TestPushReplicator.java

```diff
@@ -126,6 +148,99 @@ void packFailsWithException() throws IOException {
     output.assertClosedExactlyOnce();
   }
 
+  @Test
+  void importSameContainerWhenAlreadyImport() throws Exception {
```

Comment:
These new tests do not really test `PushReplicator`, rather `ContainerImporter`, so I would prefer if they could be added in `TestContainerImporter` (which does not exist yet).